jdaviz/configs/default/plugins/export/tests/test_export.py

```diff
@@ -205,6 +205,18 @@ def test_basic_export_subsets_cubeviz(self, cubeviz_helper, spectral_cube_wcs):
                            match='Subset can not be exported - Export for composite subsets not yet supported.'):  # noqa
             export_plugin.export()
 
+        # Test saving spectral subset
+        cubeviz_helper.app.session.edit_subset_mode.mode = NewMode
+        cubeviz_helper.app.get_viewer("spectrum-viewer").apply_roi(XRangeROI(5, 15.5))
+        export_plugin.subset.selected = 'Subset 2'
+
+        # Format should auto-update to first non-disabled entry
+        assert export_plugin.subset_format.selected == 'ecsv'
+
+        export_plugin.filename_value = "test_spectral_region"
+        export_plugin.export()
+        assert os.path.isfile('test_spectral_region.ecsv')
```

Comment:
Is roundtripping a requirement? That is, is Jdaviz expected to be able to re-ingest this file and re-creates the same spectral region?

Reply:
I don't believe it is.

Reply:
OK. I guess checking for the output content is correct or not is the job of `specutils`, so we don't have to do that here. Thanks for the clarification!